CHANGELOG.md

```diff
@@ -3,6 +3,52 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.19.10
+
+_Apr 12, 2024_
+
+We'd like to offer a big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+- 📚 Documentation improvements
+
+### Data Grid
+
+#### `@mui/x-data-grid@6.19.10`
+
+- [DataGrid] Do not escape double quotes when copying to clipboard (#12734) @cherniavskii
+- [DataGrid] Fix bug in suspense (#12754) @cherniavskii
+
+#### `@mui/x-data-grid-pro@6.19.10` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@6.19.10`.
+
+#### `@mui/x-data-grid-premium@6.19.10` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@6.19.10`.
+
+### Date Pickers
+
+#### `@mui/x-date-pickers@6.19.10`
+
+No changes.
+
+#### `@mui/x-date-pickers-pro@6.19.10` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+No changes.
```

Comment:
Just checking in: has this been agreed upon?
AFAIK, we haven't released packages in v6 if they didn't have any changes. 🤔

Reply:
I think we agreed to release everything that Lerna suggests, which is the current case :D

Reply:
I'm a bit curious why Lerna suggested releasing packages which did not receive any changes... 🤷

Reply:
No idea why :laughing: 

But we must clearly move on this discussion to have an agreement on a good release strategy

Reply:
I've switched to v6.x branch, ran `yarn install` and then `yarn release:version`.
This is the result:
<img width="813" alt="Screenshot 2024-04-12 at 16 45 08" src="https://github.com/mui/mui-x/assets/4941090/2141ed94-3b1c-4c2a-9da3-76ac1fa2c3b4">

Reply:
I did the same, even nuked by yarn cache. I can revert the other packages